spec/lib/deploy/activate_spec.rb

```diff
@@ -80,10 +80,12 @@
       expect(s3_client).to have_received(:get_object).with(
         bucket: 'login-gov.secrets.12345-us-west-1',
         key: 'common/GeoIP2-City.mmdb',
+        response_target: subject.send(:geolocation_db_path),
```

Comment:
I don't love this, but it seems like something in the test changed to be stricter during the conversion to kwargs

Reply:
Could we do one of these?
```suggestion
        response_target: kind_of(String),
```